tests/test_table.py

```diff
@@ -960,6 +968,77 @@ def test_schema_types(self, reset_db: None) -> None:
         df = t._col_descriptor()
         assert list(df['Type']) == expected_strings + expected_strings
 
+    def test_uuid_type(self, reset_db: None, reload_tester: ReloadTester) -> None:
```

Comment:
Let's move this into a new test file, test_table.py is getting a bit hard to navigate.